.github/workflows/ci.yml

```diff
@@ -55,10 +65,54 @@ jobs:
       - name: Run tests
         run: cargo test --verbose --all-targets --all-features
 
+  run-podman-tests:
+    runs-on: ubuntu-latest
+    needs: [build, build-rust]
+
+    strategy:
+      matrix:
+        test-path:
+          - ../tests/smoke/0001-smoke.zndsl
+          - ../tests/0013-db-snapshot.zndsl
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.5.0
+
+      - name: Setup containers registries
+        run: |
+          mkdir -p /etc/containers \
+          && echo "unqualified-search-registries = ['docker.io']" \
+          && sudo tee /etc/containers/registries.conf
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+
+      - name: Retrieve build artifacts
+        uses: actions/download-artifact@master
+        with:
+          name: ${{ runner.os }}-build-${{ github.sha }}
+          path: ./javascript/packages
```

Comment:
I think this should be removed as well

Reply:
Well, If I remove this, we can't use node in the job because it's started on a new runner (not a step)

Reply:
Why you need to retrieve build artifacts since we want to remove also the upload?